tests/test_auth.py

```diff
@@ -483,6 +483,62 @@ def test_lookup(self):
         eq_(session['user']['gender'], 'female')
 
 
+class TestRules(AuthBase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Another approach is to just set url = server.base_url + '/auth/rulesdb' as a class variable and not override setupClass at all. Same for TestRulesFile